### DISCUSSION TITLE
Rules on regional channels in menu?

### DISCUSSION THREAD

**Discussion by ppiotr3k** [Catch-up-TV-and-More/plugin.video.catchuptvandmore#660] [Q&A] [answered]
Dear CA TV creators and maintainers :wave::slightly_smiling_face: 

Newcomer to the project, I'm trying to contribute (context: in priority fixing and enhancing eastern EU channels due to current international context and need for people to be able to access the information of their own choice :free: :brain:), and **I have troubles to understand the rules for organizing regional channels in the menu**, due to several possible inconsistencies :thinking: 

For example, for FR channels:
* _France 3_ regional channels are handled as regional channels, i.e. using a smart `setting.language` trick to select user's region
* _BMF_ regional channels are handled each as an independent channel, i.e. being each directly in the menu, no selection
* _Vià_* regional channels are handled each as an independent channel, i.e. being each directly in the menu, no selection
(yes, most of _Vià_ channels do not exist anymore/have changed ownership and name, this is just for illustration :innocent:)

:pray: **I would therefore love to understand what are the rules to organize regional channels in the menu** :pray: 
Especially as I have several enhancements on regional channels from different countries I'd like to contribute, but I'm currently struggling with managing them as a unique channel, mostly because it seems very complex to manage associated artwork using `setting.language` :thinking: 

Thank you and kind regards! :bow:

**Comment by joaopa00** [comment, marked as answer]
Hello.
I think at the beginning of Catchuptvmore, there was no policy to decide how to implement regional language. I think France 3's trick is the way to go. Indeed, if there are ma,y channels with tons of different regions, the menu would become too heavy so unreadable..

Thanks for the remainder, I will send patches to make BFM regionals use the trick to avoid multiple items.
